### DISCUSSION TITLE
Modern Polars - 1  Indexing (Or Lack Thereof)

### DISCUSSION THREAD

**Discussion by giscus** [kevinheavey/modern-polars#4] [General]
# Modern Polars - 1  Indexing (Or Lack Thereof)

https://kevinheavey.github.io/modern-polars/indexing.html

**Comment by libcell** [comment]
it's great!

**Comment by SSSYDYSSS** [comment]
that's wonderful

**Comment by rodrigogiraoserrao** [comment]
Just to make sure I understand what's going on, in “1.4.2 Rows by string index, columns by name”, the two results look like rows have different orders.
Is it correct that Polars just filtered the rows that had the matching reporting airline whereas pandas reordered the rows of the dataframe? (When did the reordering happen? When you indexed with `loc`, right?)

**Comment by chrisgalvez** [comment]
In fact indexing is one of my favorite features in pandas. I'm not sure this is an advantage in my case, although the way of doing it in polars is not bad at all.

**Comment by jxu** [comment]
Panda's SettingWithCopyWarning is a UX crime. It has caused so much frustration and confusion over the years. And everything else (R, SQL, Polars, Spark, etc.) have no problem like it.

**Comment by willrichmond** [comment]
You can simplify the polars syntax
`df_pl.select("Dest", "Tail_Number")`
is the same as
`df_pl.select(["Dest", "Tail_Number"])`
